jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/MavenSettingsServerCredentialsTest.java

```diff
@@ -17,45 +17,78 @@
 package com.google.cloud.tools.jib.maven;
 
 import com.google.cloud.tools.jib.plugins.common.AuthProperty;
-import java.util.Arrays;
+import com.google.cloud.tools.jib.plugins.common.InferredAuthException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 import java.util.Optional;
-import org.apache.maven.settings.Server;
+import org.hamcrest.core.StringStartsWith;
```

Comment:
Huh, I've been using `org.hamcrest.CoreMatches::startsWith`. What's the preference?

Reply:
I have no preference, this was probably my first search result. Looks like `CoreMatchers` just calls into `StringStartsWith`. I changed it anyway to `CoreMatchers` for consistency.